docs/guides/Integration-Guides/erp-integration-guide/change-seller.md

```diff
@@ -52,4 +50,4 @@ This is an example of what the body of this request should look like:
 }
 ```
 
->❗ When a change of seller is successfully completed, the marketplace order loses courier information, since the array in `shippingData.logisticsInfo.deliveryIds` is now empty. In order to get the updated information, it is necessary to make a [Get order](https://developers.vtex.com/vtex-rest-api/reference/getorder) request to the new seller's endpoint, such as `https://{newSeller}.{environment}.com.br/api/oms/pvt/orders/{orderId}`.
\ No newline at end of file
+>❗ When a change of seller is successfully completed, the marketplace order loses courier information, since the array in `shippingData.logisticsInfo.deliveryIds` is now empty. In order to get the updated information, it is necessary to make a [Get order](https://developers.vtex.com/vtex-rest-api/reference/getorder) request to the new seller's endpoint, such as `https://{newSeller}.{environment}.com.br/api/oms/pvt/orders/{orderId}`.
```

Comment:
Suggestion, correct the link reference to the current address:

https://developers.vtex.com/docs/api-reference/orders-api#get-/api/oms/pvt/orders/-orderId-